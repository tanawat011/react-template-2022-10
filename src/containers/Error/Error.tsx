import { Outlet } from 'react-router-dom'

export const ErrorContainer: React.FC = () => {
  return (
    <>
      <p>ErrorContainer</p>
      <Outlet />
    </>
  )
}
