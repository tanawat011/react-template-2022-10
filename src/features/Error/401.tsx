export const Error401 = () => {
  return <div>401 Unauthorized</div>
}
