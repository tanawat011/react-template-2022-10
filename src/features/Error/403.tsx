import { PATH } from 'Routes'
import { Button } from 'components/Button'

export const Error403 = () => {
  return (
    <div>
      <p>403 Forbidden</p>

      <Button link to={`/${PATH.AUTH.ROOT}`}>
        Back to Login
      </Button>
    </div>
  )
}
