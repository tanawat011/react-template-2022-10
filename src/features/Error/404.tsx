import { useNavigate } from 'react-router-dom'

import { Button } from 'components/Button'

export const Error404 = () => {
  const navigate = useNavigate()

  return (
    <div>
      <p>404 Not Found</p>

      <Button onClick={() => navigate(-1)}>Back to Previous Page</Button>
    </div>
  )
}
