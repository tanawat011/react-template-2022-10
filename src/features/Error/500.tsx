export const Error500 = () => {
  return <div>500 Internal Server Error</div>
}
