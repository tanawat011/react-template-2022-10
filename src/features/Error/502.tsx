export const Error502 = () => {
  return <div>502 Bed Gateway</div>
}
