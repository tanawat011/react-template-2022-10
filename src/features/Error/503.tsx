export const Error503 = () => {
  return <div>503 Service Unavailable</div>
}
