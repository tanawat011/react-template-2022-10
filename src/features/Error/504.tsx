export const Error504 = () => {
  return <div>504 Gateway Timeout</div>
}
