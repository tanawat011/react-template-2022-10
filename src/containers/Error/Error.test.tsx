import { render } from '@testing-library/react'

import { ErrorContainer } from './Error'

describe('<ErrorContainer />', () => {
  test('match snapshot', () => {
    const { asFragment } = render(<ErrorContainer />)
    expect(asFragment()).toMatchSnapshot()
  })
})
