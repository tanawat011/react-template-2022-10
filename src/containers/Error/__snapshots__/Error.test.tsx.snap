// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorContainer /> match snapshot 1`] = `
<DocumentFragment>
  <p>
    ErrorContainer
  </p>
</DocumentFragment>
`;
